iep-module-aws2/src/main/java/com/netflix/iep/aws2/AwsClientFactory.java

```diff
@@ -296,19 +350,20 @@ private SdkAsyncHttpService createAsyncHttpService(String name) {
   public <T> T newInstance(String name, Class<T> cls, String accountId) {
     try {
       SdkHttpService service = createSyncHttpService(name);
+      SdkAsyncHttpService asyncService = createAsyncHttpService(name);
```

Comment:
Why was this moved up? It looks like it is only needed for the async case.

Reply:
No reason. I moved it outside when trying out some changes and forgot to clean it up when it's no longer need outside the async case. Will move it back inside the `AwsAsyncClientBuilder` case